layers/meta-balena-fsl-arm/recipes-kernel/linux/linux-fslc-imx_%.bbappend

```diff
@@ -0,0 +1,25 @@
+FILESEXTRAPATHS:append := ":${THISDIR}/${PN}"
+
+inherit kernel-resin
+
+# Latest known revision as of now is 49cd2d27fefdbaa79f32df96c4672433d9d45590
+# and is set in meta-freescale-third-party linux-boundary recipe.
+
+# Disable commit SHA in kernel version string
+SCMVERSION="n"
+
+SRC_URI:append = " \
```

Comment:
I assume these would be needed only for the machine you just defined so could we add it only for it?